packages/editor/src/index.js

```diff
@@ -1,3 +1,14 @@
+/**
+ * WordPress dependencies
+ */
+import '@wordpress/blocks';
+import '@wordpress/core-data';
+import '@wordpress/nux';
+import '@wordpress/viewport';
```

Comment:
I think `edit-post` also uses a bunch of stores, should we add theme there as well?